WordPress/Classes/ViewRelated/Activity/ActivityTableViewCell.swift

```diff
@@ -17,9 +17,9 @@ open class ActivityTableViewCell: WPTableViewCell {
 
     open func configureCell(_ activity: Activity) {
         self.activity = activity
-        timestampLabel?.attributedText = NSAttributedString(string: activity.published.mediumStringWithTime(),
+        timestampLabel?.attributedText = NSAttributedString(string: activity.published.mediumStringWithUTCTime(),
```

Comment:
Why are we showing this in UTC?

Reply:
For now to display everything as it is on the web, easier to follow and debug, we will need to convert to the site timezone in the future.